src/main/java/mate/academy/dao/impl/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,122 @@
+package mate.academy.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.ManufacturerDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Manufacturer;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacturer.manufacturers (manufacturer_name, "
+                + "manufacturer_country) VALUES (?, ?);";
```

Comment:
```suggestion
        String query = "INSERT INTO manufacturers (name, country) VALUES (?, ?);";
```
please see checklist
same for other queries

Reply:
Ok. ;)